alerts/cluster-monitoring-operator/PrometheusOperatorRejectedResources.md

```diff
@@ -0,0 +1,88 @@
+# PrometheusOperatorRejectedResources
+
+## Meaning
+
+The `PrometheusOperatorRejectedResources` alert triggers when Prometheus Operator
+rejects invalid `Alertmanager`, `PodMonitor`, `ServiceMonitor`, or `PrometheusRules`
+objects.
+
+## Impact
+
+The custom resources that trigger the alert are ignored by Prometheus Operator.
+As a consequence, they will not be part of the final configuration of the
+Prometheus, Alertmanager, or Thanos Ruler components managed by Prometheus Operator.
+
+## Diagnosis
+
+Find the type of custom resource and its namespace in the `description` field
+of the `PrometheusOperatorRejectedResources` alert. You can find this information
+by using the command line interface (CLI) or the OpenShift web console.
+
+### Diagnose using the CLI
+
+```bash
+### Gets alertmanager URL
+$ ALERT_MANAGER=$(oc get route alertmanager-main -n openshift-monitoring -o jsonpath='{@.spec.host}')
+
+### Gets fired alerts
+$ curl -k -H "Authorization: Bearer $(oc create token prometheus-k8s -n openshift-monitoring)"  https://$ALERT_MANAGER/api/v1/alerts
```

Comment:
we can filter only the `PrometheusOperatorRejectedResources` alerts + use the v2 API (v1 is deprecated).

```suggestion
$ curl -sk -H "Authorization: Bearer $(oc create token prometheus-k8s -n openshift-monitoring)"  https://$ALERT_MANAGER/api/v2/alerts?filter=alertname=PrometheusOperatorRejectedResources
```